src/main/java/net/earthcomputer/clientcommands/MultiVersionCompat.java

```diff
@@ -40,10 +43,11 @@ public boolean doesItemExist(Item item) {
     public static final MultiVersionCompat INSTANCE = Util.make(() -> {
         try {
             FabricLoader loader = FabricLoader.getInstance();
-            if (loader.isModLoaded("viafabric")) {
+            ModContainer modContainer;
+            if (loader.getModContainer("viafabric").orElse(null) != null) {
```

Comment:
why use `loader.getModContainer("viafabric").orElse(null) != null` if you dont capture the `ModContainer`?

Reply:
oh yeah right, I can probably just use isModLoaded() here

Reply:
Fixed